### DISCUSSION TITLE
Sleep Timer last minute: Fade volume, shake to reset

### DISCUSSION THREAD

**Discussion by whispersnyc** [PaulWoitaschek/Voice#2387] [Ideas]
As the title suggests, Smart Audiobook Player's sleep timer fades volume during the last minute and a persistent shake to reset voice timer feature but to save battery, only checking during the last minute may be wise.

**Comment by PaulWoitaschek** [comment]
That's exactly what voice does for years (except 10s instead of 1m fade out)

**Reply by luiandresgonzalez** [reply]
Can it be configurable? Adding to this, a feature like AntennaPod where they have auto sleep mode if playback is started after certain specified hours.

**Reply by jkarjala** [reply]
Shake does not reset the timer on my samsung S20fe, should it? 

Tapping the timer display could also reset the timer?
